packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -598,8 +598,7 @@ export const Webchat = forwardRef((props, ref) => {
               onKeyDown={e => onKeyDown(e)}
               style={{
                 display: 'flex',
-                // Disabling auto-zoom on input (iPhone devices): https://stackoverflow.com/a/25614477
-                fontSize: isIphone() ? 'initial' : 14,
+                fontSize: keyboardResizer.fontSize(),
```

Comment:
I would pass the "14" as argument to fontSize, since 14 is a general config and not something particular to the keyboardResizer

Reply:
yes, sure, would be better! added 👍